core/field_angle.ts

```diff
@@ -202,7 +202,7 @@ export class FieldAngle extends FieldTextInput {
     // #2380)
     this.symbol_ = dom.createSvgElement(Svg.TSPAN, {});
     this.symbol_.appendChild(document.createTextNode('\u00B0'));
-    this.textElement_.appendChild(this.symbol_);
+    this.getTextElement().appendChild(this.symbol_);
```

Comment:
Hey @maribethb & @BeksOmega - I went ahead and created `getTextElement` and other wrapper functions to handle encapsulating the null error check. What do you think about utilizing it in this context, though? Earlier, it was mentioned using `this.textElement_!` made sense here since it was just defined in `initView`. The trade off here I think is 'runtime compute' vs 'developer compute', so to speak 😂

Runtime Compute - In cases like this where the null check is so minor, it would take an extreme number of runs for it to impact runtime performance. 

Developer Compute - It might be obvious that after `initView` the `textElement_` should be set, though it's just one less thing a developer has to juggle when looking through this code.

Reply:
I think either option makes sense in situations like this where we never expect developers to be interacting with these properties while they're null =)

Reply:
We discussed this earlier this week and this is one of those cases where it's acceptable because it's easily verified that textElement is set directly beforehand in initView. The property is set and then used in the same method so there is no room for developer error - compare that to `fieldGroup_` where it could be set in multiple places and an error like forgetting to call an init method would result in the value being unexpectedly null.

There is no hard and fast rule I can give you but that's why this particular case the `!` is fine, though using the method is also fine.